.gitignore

```diff
@@ -4,3 +4,4 @@ vendor
 *.swp
 dist/
 *.log
+.idea/
```

Comment:
Is it normal?

Reply:
yes if somebody use goland as IDE from intelij :P

Reply:
I disagree

This should be part of user's git ignore global settings

Otherwise, a .gitignore ends with excluding a lot things because everyone has a different IDE, so one may argue there is a need for VS Code, Sublime Text.

And it looks like a sneaky change out of scope added to the PR

That said I won't fight much than this.

Reply:
@ccoVeille yes you are 100% right, I forgott about global gitignore :P

Reply:
But in general I'd reiterate the discussion separately. The purpose of `.gitignore` (in my humble opinion) is to also prevent commits by mistake. Adding such folders for popular IDEs can help. I do have a global gitignore, but that's something that doesn't come out of the box and not everyone is aware of it. Not a big deal but can save some time for reviews if someone creates a PR with such files added by mistake.